helix-term/src/config.rs

```diff
@@ -3,8 +3,10 @@ use serde::Deserialize;
 use crate::keymap::Keymaps;
 
 #[derive(Debug, Default, Clone, PartialEq, Deserialize)]
+#[serde(rename_all = "kebab-case")]
 pub struct Config {
     pub theme: Option<String>,
+    pub log_file: Option<String>,
```

Comment:
I would call this log-path rather than log file. You can also use `Option<PathBuf>` here to parse straight into a `PathBuf`.